hal_aarch64/src/lib.rs

```diff
@@ -13,6 +12,7 @@ pub mod mm;
 
 mod devices;
 
+#[allow(dead_code)]
```

Comment:
uff, why is that necessary?

Reply:
We are using debug to display this later on. But the fields are not considered as accessed so we have an unused warning which is not true.

We could put a '_' but that would make the display ugly